spec/unit/framebuffer_spec.lua

```diff
@@ -0,0 +1,30 @@
+describe("Framebuffer unit tests", function()
+    local fb
+
+    setup(function()
+        fb = require("ffi/framebuffer_dummy"):new{
```

Comment:
Oh, you forced-pushed this dummy in? There was probably overlap with #828 because iirc you used the  	framebuffer_SDL2_0 as opposed to framebuffer_SDL1_2.

I didn't investigate the switch/deprecation yet. It should be as simple as switching from libsdl1.2-dev to libsdl2-dev in the Travis setup).

That being said, a dummy might be a good idea regardless.

Reply:
Sorry, could you expand a little bit more on what you mean by overlap with #828? What do you mean by I used framebuffer_SDL2_0?

The eventual goal of this dummy module is to remove the dummy code path from SDL2.

Reply:
Iirc the initial version of this PR had a call to framebuffer_SDL2_0 which failed because that's not currently in Travis, only after which you added the dummy. Maybe I misremember? No matter, it's not important. :-)

Reply:
Ha, I see. Yeah, I replaced the SDL2 dummy call with a proper dummy module :P